coop_membership/views/view_res_partner.xml

```diff
@@ -659,7 +658,7 @@
         ])]"/>
         <field name="priority">20</field>
         <field name="arch" type="xml">
-            <xpath expr="//field[@name='sex']" position="attributes">
+            <xpath expr="//field[@name='gender']" position="attributes">
```

Comment:
This view should have inherit_id = `view_personal_information_gender`, probably
Because that's the view thats making this field readonly in some cases.
And that's the view that's actually extending `partner_contact_gender.view_personal_information_gender`, which should be the only one adding the field in the view